contrib/one_shot_models/examples/low-data-porting.ipynb

```diff
@@ -0,0 +1,405 @@
+{
```

Comment:
It might be useful to turn the working examples into unit tests so we can get them integrated into the Travis. It's also easier to debug examples once they're in unit tests :)